### PR TITLE
Add GitHub

### PR DESCRIPTION
**Pull request by mei-li**
Please review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -231,11 +231,13 @@
...
 
-Meili
+Github: mei-li
 
-meili.tr@gmail.com
+Email: meili.tr@gmail.com
 
-@meilitr
+Twitter: @meilitr
+
```
Comment (claudinos) [inline index.html:239]: Thank you for your time and the presentation about open source was very helpful to me.Claudine

File: index.html [hunk 2]
```diff
@@ -231,11 +231,13 @@
 
 # Thank you
 
-Meili
```
Comment (nicky-code) [inline index.html:234]: you have a nice name @nicky-code

File: index.html [hunk 3]
```diff
@@ -231,11 +231,13 @@
...
+Github: mei-li
 
-meili.tr@gmail.com
+Email: meili.tr@gmail.com
 
-@meilitr
+Twitter: @meilitr
+
+Thank you all you are lovely
```
Comment (Belie06Loryn) [inline index.html:240]: Thank you for sharing with us more about how we can improve our knowledge at the same time build our career.#Alexie_MANIRAKORA
Nice to meet you!

Comment (Omoyi) [inline index.html:240]: You are welcome! and you are lovely too!!!! It was nice to meet you by @Omoyi

File: index.html (hunk 1 above)
Comment (Kategaya) [inline index.html:239]: Thank you for this workshop.

File: index.html (hunk 2 above)
Comment (ritha-kankindi) [inline index.html:234]: Hello Meili,
I'm KANKINDI Ritha Veneranda and I really appreciate you for an interesting course you gave us today. it's a great contribution to my job searching journey and also empowering my career. be blessed!

File: index.html [hunk 4]
```diff
@@ -231,11 +231,13 @@
...
 # Thank you
 
-Meili
+Github: mei-li
 
-meili.tr@gmail.com
+Email: meili.tr@gmail.com
 
-@meilitr
+Twitter: @meilitr
```
Comment (mei-li) [inline index.html:238]: meilis comment

**Pull request merged**